src/lib/mina_lib/mina_lib.ml

```diff
@@ -1877,9 +1877,8 @@ let create ~commit_id ?wallets (config : Config.t) =
               | Some frontier ->
                   Transition_frontier.close ~loc:__LOC__ frontier ) ;
           (* knot-tying hacks so we can pass a get_node_status function before net, Mina_lib.t created *)
-          let net_ref = ref None in
           let sync_status_ref = ref None in
-          let get_node_status _env =
+          let get_node_status (net : Mina_networking.t) =
             O1trace.thread "handle_request_get_node_status" (fun () ->
```

Comment:
This `O1trace.thread` is inside of another in `Networking.create`, I don't know if this is used for metrics.